DietPi-Update | update medusa service

Joulinar opened the pull request:
medusa service needs to be updated as user group medusa is not existing

**Status**: Done
- [x] update `dietpi-software` to set correct value on `medusa.service`
- [x] test on VM Buster
- [x]  test on RPI Buster

**Reference**: 
- https://github.com/MichaIng/DietPi/issues/3842
- https://dietpi.com/phpbb/viewtopic.php?f=11&t=8193

**Commit list/description**:
- DietPi-Software | update `medusa.service` and remove medusa user group as it is not existing

@MichaIng 
I guess idea of the `sed` command was to set a `#` to disable `Group=medusa`. But this is not working. Medusa is still set as group and service is not starting. 

Another option would be to set `Group=dietpi` instead 
`G_CONFIG_INJECT 'Group=' 'Group=dietpi' /etc/systemd/system/medusa.service`

Joulinar committed:
DietPi-Update | update medusa service

medusa service needs to be updated as user group medusa is not existing

MichaIng commented on dietpi/dietpi-software:
```diff
@@ -10929,7 +10929,7 @@ _EOF_
 
 			# Service: https://github.com/pymedusa/Medusa/blob/master/runscripts/init.systemd
 			G_EXEC cp /mnt/dietpi_userdata/medusa/runscripts/init.systemd /etc/systemd/system/medusa.service
-			G_EXEC sed -i 's/^[[:blank:]]Group=/#Group=/' /etc/systemd/system/medusa.service
```
An asterisk is missing there, it should have been:
```
G_EXEC sed -i 's/^[[:blank:]]*Group=/#Group=/' /etc/systemd/system/medusa.service
```
This catches every possible case where an active `Group` is in place, since leading spaces are allowed in systemd units.

Joulinar replied:
ah ok, didn't know that. Still bash scripting is something fare away 🤣

MichaIng replied:
That is more about regex, `sed` and `grep` syntax 😄.

MichaIng commented:
Many thanks! A bug my end. As commented, an asterisk was intended there so that leading spaces are allowed.

MichaIng committed:
v6.34

+ DietPi-Software | Medusa: Be specific/failsafe when commenting "Group=" in systemd unit

MichaIng committed:
v6.34 (#3846)

+ Update "medusa" branch with CHANGELOG.txt v6.34 init

MichaIng committed:
v6.34

+ CHANGELOG | Medusa: Resolved an issue where Medusa failed to start after install

MichaIng commented:
Works fine here on VM and RPi, will merge.
